resources/validators.md

```diff
@@ -0,0 +1,236 @@
+---
+description: Mailbox default security settings
+---
+
+# Mailbox default security settings
+
+## Celo
+
+### Validators
+
+| Validator                                | Address                                      | Storage Location                                                                             |
```

Comment:
Optional: I think the storage locations are probably unnecessary

Reply:
why not include?

Reply:
agree "unnecessary" but maybe useful?